tajo-client/src/main/java/org/apache/tajo/client/QueryClient.java

```diff
@@ -108,17 +111,17 @@
 
   TajoMemoryResultSet fetchNextQueryResult(final QueryId queryId, final int fetchRowNum) throws SQLException;
 
-  boolean updateQuery(final String sql) throws SQLException;
+  boolean updateQuery(final String sql) throws TajoException;
 
-  boolean updateQueryWithJson(final String json) throws SQLException;
+  boolean updateQueryWithJson(final String json) throws TajoException;
 
   List<ClientProtos.BriefQueryInfo> getRunningQueryList() throws SQLException;
```

Comment:
Some of other methods in this class still throw ```SQLException```. Is it good or do you leave them for future changes?